src/guides/v2.3/howdoi/custom-attributes/text-field.md

```diff
@@ -0,0 +1,346 @@
+---
+layout: tutorial
+group: how-do-i
+subgroup:
+title: Text Field Custom Attribute
+menu_node:
+level3_subgroup: custom-attributes
+menu_order: 2
+contributor_name: Adarsh Manickam
+contributor_link: https://github.com/drpayyne
+---
+
+## Overview
+
+This tutorial describes how a developer can create a custom text field attribute for the Customer entity using code. This will reflect in both the Magento Admin's [Customer Grid](https://docs.magento.com/user-guide/customers/customer-account-manage.html) and [Customer Form](https://docs.magento.com/user-guide/customers/customer-account-update.html).
```

Comment:
```suggestion
This tutorial describes how a developer can create a custom text field attribute for the Customer entity using code. This will reflect in both the Magento Admin's [Customer Grid](https://docs.magento.com/user-guide/customers/customer-account-manage.html) and the [Customer Form](https://docs.magento.com/user-guide/customers/customer-account-update.html).
```

Reply:
Done!